src/common/components/Toast/utils/setToast.ts

```diff
@@ -0,0 +1,17 @@
+import { Bounce, toast } from "react-toastify";
+
+type TostType = "info" | "success" | "warn" | "error";
+
+export default function setToast(type: TostType, text: string): void {
+  toast[type](text, {
+    position: "bottom-center", // 위치
+    autoClose: 3000, // 시간
```

Comment:
숫자같은 것들은 numeric separator 문법을 적극적으로 활용해주시면 좋습니다!
```suggestion
    autoClose: 3_000, // 시간
```